app/src/models/collectionCard.ts

```diff
@@ -1,23 +1,29 @@
 // import { CollectionCardData } from "app/types/Collection";
 import { imageURL } from "../utils/utils";
-
+import { stringToSlug } from "../utils/utils";
+import { parseBoolean } from "../utils/utils";
 // TODO: Connect to typescript interface for CollectionCardData
 export class CollectionCardModel {
   uuid: string;
   title: string;
   url: string;
   imageID: string | null;
   imageURL: string;
-  numItems: number;
+  numberOfDigitizedItems: number;
   containsOnSiteMaterials: boolean;
 
   constructor(data: any) {
     this.uuid = data.uuid;
     this.title = data.title;
-    this.url = data.url;
-    this.imageID = data.image_id;
-    this.imageURL = imageURL(data.image_id, "full", "288,", "0");
-    this.numItems = data.numItems || 0;
-    this.containsOnSiteMaterials = data.containsOnSiteMaterials || false;
+    this.url =
+      process.env.APP_ENV === "development"
+        ? `/collections/${stringToSlug(data.title)}#/?tab=about`
```

Comment:
The `#/?tab=about` is temporary?...

Reply:
it's the default in the old app...

Reply:
https://digitalcollections.nypl.org/collections/16ad5350-c52e-012f-aecf-58d385a7bc34 works, but legacy DC always appends the `#/?tab=about`

Reply:
We talked about this offline but it should be kept to link to legacy appropriately, but once the `/collection/:slug` page is worked on in DCF, it should be removed. We also need to take into account whether there will be a tabs component in the new design that has an "About" tab.